woocommerce_fusion/tasks/sync_sales_orders.py

```diff
@@ -565,10 +565,6 @@ def create_or_link_customer_and_address(
 				if link.link_doctype == "Customer":
 					customer = frappe.get_doc("Customer", link.link_name)
 					break
-			if not customer:
```

Comment:
@AwuyaDaniel dude, you just deleted the code rather than refactoring it into a reusable function. Am I missing something?